### PR TITLE
Break out cyhy-core install and retrieval of GitHub OAuth token into roles

### PR DESCRIPTION
**Pull request by jsf9k**
Both these tasks are repeated in the Ansible code, so it makes sense to break them out into their own roles.

### CODE REVIEW

File: packer/ansible/roles/cyhy_core/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  galaxy_tags: []
+    # List tags for your role here, one per line. A tag is a keyword that describes
+    # and categorizes the role. Users find roles by searching for tags. Be sure to
+    # remove the '[]' above, if you add tags to this list.
+    #
+    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
+    #       Maximum 20 tags per role.
+
+dependencies: []
```
Comment (dav3r) [inline packer/ansible/roles/cyhy_core/meta/main.yml]: @jsf9k Should we include github_oauth as a role dependency here?
https://docs.ansible.com/ansible/2.5/user_guide/playbooks_reuse_roles.html#role-dependencies

**Review by dav3r — APPROVED** [review]
Looks solid @jsf9k - I only have that one comment about dependencies.

File: packer/ansible/roles/cyhy_core/meta/main.yml (hunk 1 above)
Comment (jsf9k) [reply packer/ansible/roles/cyhy_core/meta/main.yml]: Thanks for pointing that out @dav3r!  I will go ahead and add `github_oauth` as a dependency where appropriate, and I will create another PR where I take full advantage of dependencies to simplify our Ansible code.

**Comment by jsf9k** [comment]
@dav3r, please take another look.  If you're still happy then I'll merge this pull request and start applying dependencies globally.

**Review by dav3r — APPROVED** [review]
Looks great @jsf9k - nice work!

**Pull request merged**